crypto/store/loader_file.c

```diff
@@ -504,7 +509,7 @@ static OSSL_STORE_INFO *try_decode_params(const char *pem_name,
                 && (ameth = EVP_PKEY_get0_asn1(pkey)) != NULL
                 && ameth->param_decode != NULL
                 && ameth->param_decode(pkey, &tmp_blob, len)) {
-                (*matchcount)++;
+                *matchcount = 1;
```

Comment:
This change is not correct.  It *should* count every match, so the calling function can correctly report ambiguity.